city_scrapers/spiders/cook_emergency_telephone.py

```diff
@@ -0,0 +1,135 @@
+import re
+from datetime import datetime, timedelta
+from io import BytesIO
+
+import scrapy
+from city_scrapers_core.constants import FORUM
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from PyPDF2 import PdfFileReader
+
+
+class CookEmergencyTelephoneSpider(CityScrapersSpider):
+    name = "cook_emergency_telephone"
+    agency = "Cook County Emergency Telephone System Board"
+    timezone = "America/Chicago"
+    start_urls = ["https://www.cookcounty911.com/"]
+    location = {
+        "name": "Conference Room",
+        "address": "1401 S. Maybrook Drive, Maywood, IL 60153",
+    }
+
+    def __init__(self, *args, **kwargs):
+        self.meeting_starts = []
+        self.docs_link = ""
+        self.agenda_link = ""
+        super().__init__(*args, **kwargs)
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        schedule_link = ""
+        for link in response.css(".sliding_box a"):
+            link_text = link.css("*::text").get()
+
+            schedule_link = self.start_urls[0] + '/wp-content/uploads/pdfs/'
+
+            if "Minutes" in link_text:
+                self.docs_link = link.attrib["href"]
+            elif "Agenda" in link_text:
+                self.agenda_link = link.attrib["href"]
```

Comment:
Since they're always using the same URL for agendas even though the file changes, we should include a query parameter in the URL so that there's a way of distinguishing between copies (since this will be archived to the Wayback Machine either way). We're taking this approach for the Chicago Board of Election Commissioners:

https://github.com/City-Bureau/city-scrapers/blob/db6cba7919bd6450e8e74fb4cd2cceea135f5184/city_scrapers/spiders/chi_board_elections.py#L149

Reply:
Ok I will add it. But if someone visits this link, they will still see the upcoming agenda only, right?

Reply:
In general yes, but we're archiving everything so if it's stored when it's first generated we can link either to the Wayback Machine or a saved copy